user-contracts-mon/internal/parser.go

```diff
@@ -0,0 +1,45 @@
+package monitor
+
+import (
+	"errors"
+	"os"
+	"strconv"
+	"strings"
+
+	env "github.com/hashicorp/go-envparse"
+)
+
+var invalidCfgError = errors.New("Invalid or Missing Fields in configration file")
+
+func parseFile(envPath string) (string, error) {
```

Comment:
parse file only reads the file's contents, right?
i mean is there any parsing envolved?